sos/collector/__init__.py

```diff
@@ -607,12 +614,16 @@ def write_host_group(self):
         cfg = {
             'name': self.opts.save_group,
             'master': self.opts.master,
-            'cluster_type': self.cluster_type,
+            'cluster_type': self.cluster.cluster_type[0],
             'nodes': [n for n in self.node_list]
         }
-        if not os.path.isdir(COLLECTOR_LIB_DIR):
-            raise OSError("%s no such directory" % COLLECTOR_LIB_DIR)
-        fname = COLLECTOR_LIB_DIR + '/' + cfg['name']
+        if os.getuid() != 0:
```

Comment:
Ditto on another place why `root` cant have its own `.config` (but again, I am ok with the current behaviour).